frontends/ol-components/src/components/Logo/Logo.tsx

```diff
@@ -8,7 +8,7 @@ type PlatformObject = {
 
 export const PLATFORMS: Record<PlatformEnum, PlatformObject> = {
   [PlatformEnum.Ocw]: {
-    name: "OCW",
+    name: "MIT OpenCourseware",
```

Comment:
I'll triple-check with OCW, but I've never seen it with a lower-case w
```suggestion
    name: "MIT OpenCourseWare",
```

Reply:
I just did it that way because that's the way it's written in the issue, but if the W should be capitalized I'll do that.